dev-util/bats/bats-1.4.1.ebuild

```diff
@@ -20,7 +20,11 @@ RDEPEND="${DEPEND}"
 S="${WORKDIR}/${MY_PN}-${PV}"
 
 src_test() {
-	bin/bats --tap test || die "Tests failed"
+	local my_jobs=$(makeopts_jobs)
+	if ! command -v parallel; then
+		my_jobs=1
+	fi
+	bin/bats --tap --jobs "$my_jobs" test || die "Tests failed"
```

Comment:
can't but to think this is overcomplicating something simple, but my brains don't bend to this right now. Seems to work. I'll bash-it-out while merging, thanks!